app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,24 @@
+class ProjectsController < ApplicationController
+    before_action :authenticate_user! # Add this line if using Devise for authentication
+  
+    def index
+      @projects = current_user.projects # Show projects managed by the current user
+    end
+  
+    def new
+      @project = Project.new
+    end
+  
+    def create
+      @project = current_user.projects.build(project_params)
+  
+      if @project.save
+        redirect_to projects_path, notice: 'Project was successfully created.'
```

Comment:
Use translation. Move hardcoded strings to translation file

Reply:
ok boss